<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" hash="2259019727">
    <data-source source="LOCAL" name="Django default" uuid="d3133722-6890-4939-930b-5432c2edfa32">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/lightit_test/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\Python\web\venv\lightit\lightit_test\db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/xerial-sqlite-license.txt</url>
        </library>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/sqlite-jdbc-3.8.11.2.jar</url>
        </library>
      </libraries>
    </data-source>
  </component>
</project>